packages/core/src/access/index.ts

```diff
@@ -0,0 +1,20 @@
+import { BaseListTypeInfo } from '../types';
+import { BaseAccessArgs } from '../types/config/access-control';
+
+export function allowAll() {
+  return true;
+}
+export function denyAll() {
+  return false;
+}
+
+export function allOperations<ListTypeInfo extends BaseListTypeInfo>(
+  func: (args: BaseAccessArgs<ListTypeInfo>) => boolean
```

Comment:
```suggestion
  func: (args: BaseAccessArgs<ListTypeInfo> & { operation: Operation }) => MaybePromise<boolean>
```

Reply:
(might want to name and export the intersected type from the access control config types)